### ISSUE TITLE
Process.signal_kill generate zombie process

### ISSUE THREAD

**Issue by SewerynKaminski** [vlang/v#20668]
### Describe the bug

Using siganl_kill() to terminate proces leave zombie process in the system

### Reproduction Steps

call p.siganl_kill()
call p.wait()
and observe in htop zombie processes

### Expected Behavior

No zombie processes

### Current Behavior

Zombie processes

### Possible Solution

To fix it call os.execute('kill $pid') insted of the signal_kill()

### V version

V  0.4.4 f31cf6a

### Environment details (OS name and version, etc.)

Linux Ubuntu 22.04
> [!NOTE]
> You can use the 👍 reaction to increase the issue's priority for developers.
>
> Please note that only the 👍 reaction to the issue itself counts as a vote.
> Other reactions and those to comments will not be taken into account.

**Comment by JalonSolov** [comment]
Ctrl-C usually sends SIGINT, not SIGKILL.  Did you try that?

**Comment by SewerynKaminski** [comment]
> Ctrl-C usually sends SIGINT, not SIGKILL. Did you try that?

os.execute('kill $pid')  sends TERM(15) signal and it's OK (no zmbie process)
os.execute('kill -s 2 $pid')  sends INT(2) signal and it's OK (no zmbie process)

**Comment by SewerynKaminski** [comment]
Case 1

```
C.kill ( pid, int ( os.Signal.term ) ) // p.status==.running
	 // p.status==.running
p.wait() // work as expected
```
`p._wait()` is called in `pub fn (mut p Process) wait()`

Case 2
```
p.signal_kill() // p.status==.running
	 // p.status==.aborted
p.wait() // don't work as expected
```
`p._wait()` is not called in `pub fn (mut p Process) wait()`

**Comment by spytheman** [comment]
Can that be closed, since #20671 is now merged?
